Tropos/Models/DailyForecast.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+@objc(TRDailyForecast) class DailyForecast: NSObject {
+    let date: NSDate
+    let conditionsDescription: String
+    let highTemperature: Temperature
+    let lowTemperature: Temperature
+
+    init?(JSON: AnyObject?) {
```

Comment:
Answering my own question with a semi question: Because it was Objective-C?

Is it worth considering changing this to Argo now that it's Swift?